model.py

```diff
@@ -3151,6 +3115,29 @@ def with_no_genres(self, q):
         q = q.filter(WorkGenre.genre==None)
         return q
 
+    @classmethod
+    def from_identifiers(cls, _db, identifiers, base_query=None):
+        """Finds all of the works indicated by a list of identifiers"""
```

Comment:
Make the rules clear in the docstring. A work is indicated by an identifier if one of its LicensePools is directly associated with the identifier, or if one of its LicensePools is directly equivalent to the identifier.